cmd/admin-handlers.go

```diff
@@ -71,6 +71,37 @@ const (
 	mgmtForceStop   = "forceStop"
 )
 
+func getABinaryPerArchitecture(w http.ResponseWriter, r *http.Request, arch string) (crTime time.Time, releaseInfo string, lrtime time.Time, sha256Sum []byte, uReturn *url.URL, returnReader io.Reader, err error) {
+	architecture := "https://dl.min.io/server/minio/release/" + arch + "/minio.sha256sum"
```

Comment:
We already have this @cniackz 

```
updateURL := minioReleaseURL + "minio.sha256sum"
```


You do not need any `arch` input since a given binary can only be upgraded to the same arch that it is running as its not a variable.